tests/ert/unit_tests/config/test_forward_model.py

```diff
@@ -961,3 +961,18 @@ def validate_pre_experiment(self, fm_step_json: ForwardModelStepJSON) -> None:
             FORWARD_MODEL FMWithFMStepValidationError
             """
         )
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+def test_that_required_keywords_in_forward_model_are_validated():
+    Path("job").write_text("EXECUTABLE echo\nREQUIRED MESSAGE", encoding="utf-8")
+    with pytest.raises(
+        ConfigValidationError, match="Required keyword MESSAGE not found"
+    ):
+        ErtConfig.from_file_contents(
+            """
+           NUM_REALIZATIONS 1
+           INSTALL_JOB job job
```

Comment:
Let us use "step" instead of "job" to aid erasing our muscle memory on the usage of this term.
```
INSTALL_JOB step step
FORWARD_MODEL step
```

Reply:
Fixed